server/models/models.py

```diff
@@ -0,0 +1,42 @@
+from server import db, flask_bcrypt, app
+import datetime
+import jwt
+
+
+class User(db.Model):
+    __tablename__ = "users"
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    email = db.Column(db.String(255), unique=True, nullable=False)
+    password = db.Column(db.String(255), nullable=False)
+    registered_time = db.Column(db.DateTime, nullable=False)
+
+    def __init__(self, email, password, registered_time):
+        self.email = email
+        self.password = flask_bcrypt.generate_password_hash(password, app.config.get("BCRYPT_LOG_ROUNDS")).decode()
+        self.registered_time = registered_time
```

Comment:
It doesn't look like you're passing registered_time when you create a user in RegisterAPI. Should the time be set here in __init__ instead?